src/classes/BDI_FieldMappingCustomMetadata.cls

```diff
@@ -104,6 +104,16 @@ public with sharing class BDI_FieldMappingCustomMetadata implements BDI_FieldMap
         return fieldMappingInstance;
     }
 
+    /**
+    * @description Sets instance member variable holding instance of this class.  Primarily
+    * used to facilitate dependency injection in tests.
+    * @param bdiFieldMappingInstance An instance of the BDI_FieldMappingCustomMetadata class.
+    */
+    @TestVisible
+    private static void setInstance(BDI_FieldMappingCustomMetadata bdiFieldMappingInstance){
```

Comment:
It could be, but we might want to think through whether we want BDI_FieldMapping to be an interface or an abstract class if we do that.  And since we just submitted a refactor of these classes, I kept it this way for now - though we might want to revisit again after the refactor and this work is merged.